src/Pages/EventCardDetails/EventDetails.jsx

```diff
@@ -0,0 +1,288 @@
+                <div className="d-flex flex-column px-2 flex-grow-1 ">
+                  <div
+                    className={`w-100 h-100  position-relative ${styles.event_tile}`}
+                    style={{
+                      background: `url(${eventDetails?.backdrop})`,
+                    }}
+                  >
+                    <div
+                      className={` d-flex pt-5 flex-column px-5 pb-3 justify-content-end w-100 h-100 ${styles.overlay}`}
+                    >
+                      <div className="text-white">
+                        {eventDetails?.tags.map((tag, ind) => (
+                          <Tag key={`tag-${ind}`} disabled tag={tag} />
+                        ))}
+                        <h1 className="mb-2  main_font">
+                          {eventDetails?.name}
+                        </h1>
+                        <p className=" main_font">{eventDetails?.desc}</p>
+                      </div>
+                    </div>
+                    {eventDetails?.liveStreamVideoId && (
+                      <>
+                        <Button
+                          className="animate__animated animate__fadeIn position-absolute top-0 end-0 br-20p m-3 px-3 py-2 main_font rounded border border-dark"
+                          variant="danger"
+                          size="sm"
+                          onClick={() => {
+                            setVideoOpen(true);
+                          }}
+                        >
+                          <FontAwesomeIcon
+                            icon={faTelevision}
+                            color="white"
+                            size="1x"
+                            className="me-2 fa-fw"
+                          />
+                          Watch Live!
+                        </Button>
+
+                        <ModalVideo
+                          channel="youtube"
+                          isOpen={isVideoOpen}
+                          videoId={eventDetails?.liveStreamVideoId}
+                          youtube={{
+                            mute: 1,
+                            autoplay: 1,
+                          }}
+                          onClose={() => setVideoOpen(false)}
+                        />
+                      </>
+                    )}
+                  </div>
+                </div>
+                <div className="mt-3 d-flex flex-row flex-wrap">
+                  <div
+                    className={`col-12 px-2 mb-3 ${
+                      eventDetails?.isRegistrationOpen && "col-xl-8"
+                    }`}
+                  >
+                    <EventDetailsTile
+                      buttonColor="warning"
+                      buttonHandler={() => {
+                        const url = eventDetails?.rulebookUrl;
+                        if (url) window.open(`//${url}`, "_blank");
+                      }}
+                      buttonText="Know More"
```

Comment:
can you change this to "Rulebook", as in the event card tile also we have know more, its looking repetitive